### PR TITLE
Adds BookSnapshot backend to store the latest orderbook snapshot in a redis key

### PR DESCRIPTION
**Pull request by nikalexis**

### CODE REVIEW

**Comment by bmoscon** [comment]
@winnerNick let me test this out and I'll get back to you. sorry for the delay

**Comment by winnerNick** [comment]
Are you gonna merge this please?
It's an easy and I guess useful patch.

Thanx!

File: cryptofeed/backends/redis.py [hunk 1]
```diff
@@ -113,6 +128,16 @@ def __init__(self, *args, snapshots_only=False, snapshot_interval=1000, **kwargs
         super().__init__(*args, **kwargs)
 
 
+class BookSnapshot(RedisKeyCallback, BackendBookCallback):
```
Comment (bmoscon) [inline cryptofeed/backends/redis.py:131]: Can you rename this something that reflects its Redis/Redis K-V only?

File: examples/demo_redis.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 associated with this software.
 '''
 from cryptofeed import FeedHandler
-from cryptofeed.backends.redis import BookRedis, BookStream, CandlesRedis, FundingRedis, OpenInterestRedis, TradeRedis
+from cryptofeed.backends.redis import BookRedis, BookStream, CandlesRedis, FundingRedis, OpenInterestRedis, TradeRedis, \
+    BookSnapshot
```
Comment (bmoscon) [inline examples/demo_redis.py:9]: I like long lines, can you make this one line please?

**Review by bmoscon — CHANGES_REQUESTED** [review]

**Comment by bmoscon** [comment]
@winnerNick If you address the two comments I just left then, yes, I will merge it.

File: cryptofeed/backends/redis.py (hunk 1 above)
Comment (winnerNick) [reply cryptofeed/backends/redis.py:131]: Sorry, I don't get it but you can tell me a name that match for you to rename it.

Comment (bmoscon) [reply cryptofeed/backends/redis.py:131]: BookSnapshotRedis should be fine

**Comment by nikalexis** [comment]
Recommended changes done.

**Review by bmoscon — APPROVED** [review]

**Pull request merged**